assets/css/admin.rtl.css

```diff
@@ -131,6 +131,14 @@
 	background-image: url( '../images/payment-methods/afterpay-icon.svg' );
 }
 
+.payment-method__brand--afterpay_clearpay.account-country--gb {
```

Comment:
This has been nagging me for a while, as it was auto-generated on build but not committed for some reason.